### PR TITLE
Update Bootstrap version

### LINKED ISSUE #125: Update to Bootstrap 4 beta
Right now, we're using Bootstrap 4 alpha 6. We need to consider what changes are required to do this upgrade and/or track what can be a blocker.

### PR DESCRIPTION
**Pull request by chocoelho**
## Description

This PR updates supported Bootstrap 4 version from alpha to beta. `normalize` was removed from beta version, so we don't need that in `.bootstraprc` anymore.

## Motivation and Context

Resolves #125. We upgrade from alpha to beta version of Bootstrap 4.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires documentation updates.
- [x] I have updated the documentation accordingly.
- [x] My change requires dependencies updates.
- [x] I have updated the dependencies accordingly.

### CODE REVIEW

**Comment by chocoelho** [comment]
Thanks, @aericson. I also noticed that I had forgotten to include Popper.js configuration in webpack.

As to what you said:
* Removed responsive-embed: true from .bootstraprc-4-default
  * I just removed that, even though it was set to `false`, but that's not working anymore
* Removed useFlexbox: true and all mentions/documentation of it. We use useFlexbox: false. Does that still work? I think this won't work because of this change. If so, how can we disable it and do we still have a valid reason to disable it? Maybe @laisvarejao can help.
  * They removed `useFlexbox` option since BS4 uses flexboxes by default. I don't know if that's something we can change right now.
* They changed the order of sass-loader and postcss-loader in their example. They might have had a good reason to do it.
  * Basically, because order matters. Thanks for noticing that, I fixed it.

**Comment by laisvarejao** [comment]
@aericson where did you see that the order of the sass-loader and postcss-loader changed?
I don't see this in the example:
https://github.com/shakacode/bootstrap-loader/blob/master/.bootstraprc-4-default

@aericson @chocoelho yes, Flexbox is now the only option! This has changed since Alpha 6:
> Flexbox is now on by default, with no fallback! This means we've dropped IE9 support, but with significant savings to our code base, simpler components, and improved customization thanks to the power of flexbox. (https://github.com/twbs/bootstrap/issues/20939)

- I've also noticed they are alphabetically ordering the styles components in `bootstrap.rc`. I think this is a good standard to follow.

**Comment by chocoelho** [comment]
@laisvarejao about the changed order: https://github.com/shakacode/bootstrap-loader/pull/316#issuecomment-322984785

**Comment by chocoelho** [comment]
Regarding the alphabetic order, that can be done as well and I agree with it.
